extensions/amp-position-observer/0.1/test-e2e/test-target-id.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```

Comment:
If you submit this in the new year, update the date :tada:

Reply:
Happy new year :smiley:

Reply:
2020-ified!

Reply:
Woot woot!